README.md

```diff
@@ -19,11 +22,14 @@ header in the composer (To/Cc/Bcc), adding an attachment, hide the
 folder view.
 
 
+Modified by opto (Klaus Buecher) for TB 68 in https://github.com/opto/nostalgy
```

Comment:
After merging the PR, the official repo would still be github.com/nostalgy/nostalgy.

Reply:
@opto Same here. The conversation is marked as resolved, but I can see no change to the code.